packages/models/src/errors.ts

```diff
@@ -131,6 +134,10 @@ const errorCodes = {
 export type CommonErrorCodes = keyof typeof errorCodes;
 
 export function parseErrorMessage(error: unknown): string {
+  if (error instanceof ZodError) {
+    return error.errors.map((e) => fromZodIssue(e).message).join(" ; ");
+  }
```

Comment:
There is also a `fromZodError` method in `zod-validation-error` that makes a `ZodError` human readable without having to manually map and join: https://github.com/causaly/zod-validation-error?tab=readme-ov-file#fromzoderror